### DISCUSSION TITLE
Merge Editor Accept Merge Button not displaying in result editor

### DISCUSSION THREAD

**Discussion by inchara-1998** [microsoft/vscode-discussions#265] [Extension Development QnA] [answered]
I am working on merge editor : to open editor i am executiong below command.
vscode.commands.executeCommand('_open.mergeEditor', options);
But in Result editor i am not able to add "Accept Merge" button.
I am trying to add button through "editor/content" in package.json
But it requires Proposed Api to enable it everytime.

Is there any way i can add button in my result editor??

vs code version :1.73.1

**Comment by isidorn** [comment, marked as answer]
I believe this is currently not possible.
fyi @hediet
